src/components/Input/Input.tsx

```diff
@@ -81,11 +92,18 @@ class Input extends UIComponent<any, any> {
     return null
   }
 
+  handleIconOverrides = predefinedProps => {
+    return {
+      tabIndex: this.computeTabIndex,
+    }
+  }
+
   renderComponent({ ElementType, classes, rest }) {
     const { children, className, icon, input, type } = this.props
     const [htmlInputProps, restProps] = this.partitionProps()
 
     const inputClasses = cx(classes.input)
+    const iconClasses = cx(classes.icon)
```

Comment:
No need for using cx here, as the classes are already calculated.

Reply:
fixed